src/Accounts/AssemblyLoading/ConditionalAssemblyProvider.cs

```diff
@@ -43,14 +43,14 @@ public static void Initialize(string rootPath, IConditionalAssemblyContext conte
                 // todo: consider moving the list to a standalone config file
                 #region AssemblyList
                 CreateAssembly("netcoreapp2.1", "Azure.Core", "1.35.0.0").WithPowerShellCore(),
-                CreateAssembly("netcoreapp2.1", "Microsoft.Identity.Client", "4.49.1.0").WithPowerShellCore(),
-                CreateAssembly("netcoreapp3.1", "Microsoft.Identity.Client.Extensions.Msal", "2.23.0.0").WithPowerShellCore(),
```

Comment:
We don't need to load different assemblies for different platforms anymore?

Reply:
> We don't need to load different assemblies for different platforms anymore?

We use assembly for .net 2.0 as the replacement. Test it OK.